python/ray/autoscaler/aws/example-full.yaml

```diff
@@ -1,10 +1,6 @@
 # An unique identifier for the head node and workers of this cluster.
 cluster_name: default
 
-# The minimum number of workers nodes to launch in addition to the head
-# node. This number should be >= 0.
-min_workers: 0
-
 # The maximum number of workers nodes to launch in addition to the head
 # node. This takes precedence over min_workers.
```

Comment:
```suggestion
# node.
```

This should be a case of failure not precedence right?

Reply:
> This should be a case of failure not precedence right?

I'm guessing that comment describes the behavior in this line:
https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/_private/autoscaler.py#L422

I know we validate that global max_workers is at least summed node type min_workers as of https://github.com/ray-project/ray/pull/13811

But I don't think node-type max_workers < min_workers currently leads to failure -- it just forces behavior equivalent to
max_workers = min_workers.

Reply:
Uh that sentence just needs to be removed because there isn't a global min_workers anymore.